db/migrate/20181025205222_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :genre, :null => false, :default => ""
```

Comment:
remember the preffered style for Hash of symbol keys is `t.string :genre, null: false, default: ""`